appinfo/info.xml

```diff
@@ -35,7 +35,7 @@ To more informations about how this app works or to get examples, look at the ad
 	<bugs>https://github.com/librecodecoop/l10n_override/issues</bugs>
 	<repository type="git">https://github.com/librecodecoop/l10n_override</repository>
 	<dependencies>
-		<nextcloud min-version="28" max-version="28"/>
+		<nextcloud min-version="30" max-version="31"/>
```

Comment:
Another point....

I know that could be a bit complex to make backports but we can set the min and max versionto 32 and after this, send the backport to a branch stable31.

We also can drop Nextcloud 30 because the official support by server will finish into September as we can see here: https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule

Reply:
Is there any upside to going the backport way ?
I think normally this would be done when code changes to the app are needed between the two versions, so features can be backported to the old branch too.
Since this is only compatibility changes, no feature changes and no code changes between the versions are needed I think this approach would be simpler:
Releasing one version which covers both 30 and 31 and just dropping NC 30 with the a next release after September, when it is no longer supported.